test/integration/admin_user_creates_item_test.rb

```diff
@@ -0,0 +1,38 @@
+require "test_helper"
+
+class AdminUserCreatesItemTest < ActionDispatch::IntegrationTest
+  test "admin creates a item" do
+    admin = create(:admin)
+    artist = create(:artist)
+    image_path = "https://petenelson.com/wp-content" \
+                 "/uploads/2011/08/ron-swanson-meat.jpg"
+
+    category1, category2, category3 = create_list(:category, 3)
```

Comment:
Useless assignment to variable - `category2`. Use `_` or `_category2` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `category3`. Use `_` or `_category3` as a variable name to indicate that it won't be used.